code/world.dm

```diff
@@ -71,7 +71,8 @@ var/list/map_transition_config = MAP_TRANSITION_CONFIG
 	map_name = "Unknown"
 	#endif
 
-
+	for (var/typepath in (typesof(/datum/menu)-/datum/menu))
```

Comment:
@Cyberboss that would break the game.

byond bug, the world definition has to be before the first datum definition or world init order completely changes and everything depending on the old behavior breaks